client/src/components/PublicLobbyList.tsx

```diff
@@ -114,7 +114,7 @@ export function PublicLobbyList(props: PublicLobbyListProps) {
                       className={"mt-2"}
                       onClick={() => {
                         if (
-                          lobby.state &&
+                          !lobby.state ||
```

Comment:
This was needed because lobbyState is only set after the first player joins, so if it is null, we can assume no one has joined yet